packages/protocol/deployments/mainnet-contract-logs.md

```diff
@@ -150,25 +152,26 @@
   - prover_set: ``
   - proposer_one: `0xd8dA6BF26964aF9D7eEd9e03E53415D37aA96045` vitalik.eth
   - proposer: `0x000000633b68f5d8d3a86593ebb815b4663bcbe0`
+  - chain_watchdog: `0xE3D777143Ea25A6E031d1e921F396750885f43aC`
 - logs:
   - deployed on May 1, 2024 @commit `56dddf2b6`
   - admin.taiko.eth accepted the ownership @tx`0x0ed114fee6de4e3e2206cea44e6632ec0c4588f73648d98d8df5dc0183b07885`
   - Upgraded from `0xd912aB787624c9eb96a37e658e9596e114360440` to `0xF1cA1F1A068468E1dcF90dA6add185467de80943` @commit `b90b932` @tx`0x416560cd96dc75ccffebe889e8d1ab3e08b33f814dc4a2bf7c6f9555071d1f6f`
   - `Init2()` called @tx `0x7311fee56f87294e336393b55939489bc1e810c402f304013475d04c90ca32a9`
 - todo:
   - register `prover_set`
+  - register `chain_watchdog` on May 21 @tx ``
```

Comment:
```suggestion
  - register `chain_watchdog` on May 21 @tx `0xaed098ad0c93113e401f61358f963501f40a046c5b5b659a1610f10120a9a86b`
```

Reply:
we should remove `chain_watchdog` registration until we tested the feature in hekla.

We can do it with the Bridge upgrade.

Reply:
added a todo